job-server/src/main/scala/db/postgresql/migration/V0_7_4/V0_7_4__Migrate_Blobs.scala

```diff
@@ -0,0 +1,72 @@
+package db.postgresql.migration.V0_7_4
+
+import java.sql.Connection
+
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.TimeoutException
+import scala.concurrent.duration.DurationInt
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration
+import org.slf4j.LoggerFactory
+
+import javax.sql.rowset.serial.SerialBlob
+import slick.dbio.DBIO
+import slick.driver.PostgresDriver.api.actionBasedSQLInterpolation
+import slick.jdbc.GetResult
+import slick.jdbc.PositionedParameters
+import slick.jdbc.SetParameter
+import spark.jobserver.slick.unmanaged.UnmanagedDatabase
+import java.sql.Blob
+import scala.util.control.NonFatal
+
+class V0_7_4__Migrate_Blobs extends JdbcMigration {
+  private val Timeout = 10 minutes
+  private val logger = LoggerFactory.getLogger(getClass)
+
+  private case class BinaryContent(id: Int, binary: Blob)
+
+  private def logErrors = PartialFunction[Throwable, Unit] {
+    case e: Throwable => logger.error(e.getMessage, e)
+  }
+
+  private def insertBlob(db: UnmanagedDatabase, b: BinaryContent): Unit = {
+    implicit object SetSerialBlob extends SetParameter[SerialBlob] {
+      def apply(v: SerialBlob, pp: PositionedParameters) {
+        pp.setBlob(v)
+      }
+    }
+    val blob = new SerialBlob(b.binary.getBytes(1, b.binary.length().toInt))
+    val insertBlob = sqlu"""INSERT INTO "BINARIES_CONTENTS" ("BIN_ID", "BINARY") VALUES (${b.id}, ${blob})"""
```

Comment:
I wish there was a way to unify these different migrations for different DBs together.  It seems most of the code is really identical, other than the SQL statements are a tiny bit different.